portal/src/components/DatesRange/DatesRange.component.js

```diff
@@ -12,29 +19,23 @@ export default {
     this.popupItem = this.$el
   },
   data() {
-    const { value } = this
+    const { datesFilter } = this
```

Comment:
Waarom heb je dit veranderd? Kunnen we niet de value blijven gebruiken? Hoe werkt die `$emit` nu? Die werkt toch met v-model?

Reply:
Ja, lijkt inderdaad kapot. Zie comments

Reply:
het is zeker niet kapot, het werkt nu met props.